### DISCUSSION TITLE
indices of maximum values in a 3D medical image

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#5841] [Q&A] [answered]
Thanks for the great work. In my own project, I want to find the indices of the maximum value in a 3D medical image, how to achieve this?

**Comment by ebrahimebrahim** [comment, marked as answer]
If your image is some kind of array (numpy, torch tensor, etc.) then `argmax` is what you're looking for.

**Reply by YingdiZhang** [reply]
Thanks for your reply, argmax returns indices along one dim, while I want the indices of the maximum value in each dim

**Reply by ebrahimebrahim** [reply]
You can pass a `dim` (torch) or `axis` (numpy) argument to get the maximum value along a particular dim or a tuple of dims.
See
- https://pytorch.org/docs/stable/generated/torch.argmax.html
- https://numpy.org/doc/stable/reference/generated/numpy.argmax.html
